cocos/scene-graph/prefab/prefab-info.ts

```diff
@@ -202,7 +202,7 @@ export class PrefabInfo {
 
     @serializable
     @type([TargetOverrideInfo])
-    public targetOverrides?: TargetOverrideInfo[];
+    public targetOverrides?: TargetOverrideInfo[] = [];
```

Comment:
The type is `targetOverrides?`, why need a default value?

Reply:
Revisions have been rolled back, by has issue：https://github.com/cocos/3d-tasks/issues/14887